packages/react-core/src/components/Card/Card.tsx

```diff
@@ -14,15 +14,21 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   component?: keyof JSX.IntrinsicElements;
   /** Modifies the card to include compact styling. Should not be used with isLarge. */
   isCompact?: boolean;
-  /** Modifies the card to include selectable styling */
+  /** Flag indicating that the card is selectable. */
   isSelectable?: boolean;
   /** @deprecated Specifies the card is selectable, and applies raised styling on hover and select */
   isSelectableRaised?: boolean;
-  /** Modifies the card to include selected styling */
-  isSelected?: boolean;
-  /** Modifies the card to include clickable styling */
+  /** Flag indicating that the card is clickable and contains some action that triggers on click. */
   isClickable?: boolean;
-  /** Modifies a clickable or selectable card to have disabled styling. */
+  /** Flag indicating whether a card that is both clickable and selectable is currently selected and has selected styling.
+   * This will not determine the card's actual selected state.
+   */
+  isSelected?: boolean;
```

Comment:
We may want to deprecate this prop and come up with a better name.  The prop not determining the selected state is confusing given the name.

Reply:
True. We could keep this as is and I can update the logic so that isSelected determines the `isChecked` value within CardHeader (essentially what I'm doing with the isClicked prop for "clickable" cards). That would just require deprecating the `isChecked` prop in CardHeader's `CardHeaderSelectableActionsObject` interface. Doing that would be a little more consistent with the API with this update; rather than needing to pass a prop in CardHeader tor selectable cards and a prop on Card for clickable cards. WDYT?

Reply:
Yes, that sounds like it would be more intuitive for the consumer.